src/core/sys/posix/sys/socket.d

```diff
@@ -1829,7 +1829,7 @@ else version( CRuntime_Bionic )
 else version( CRuntime_Musl )
 {
     alias uint socklen_t;
-    alias ubyte sa_family_t;
+    alias ushort sa_family_t;
```

Comment:
See the header upstream: https://git.musl-libc.org/cgit/musl/tree/include/alltypes.h.in#n71